neuralmagic/tools/profiler/visualize_trace.py

```diff
@@ -43,30 +383,36 @@ def shorten_plot_legend_strings(legend, max_char_len: int):
         type=str,
         required=True,
         help="json trace file output by examples/offline_profile.py")
-    parser.add_argument(
-        "--output",
-        type=str,
-        required=False,
-        help="Output figure file, should be a image file such as pdf, "
-        "jpeg, png, etc., defaults to <json_trace>.pdf")
+    parser.add_argument("--output-directory",
+                        type=str,
+                        required=False,
+                        help="Directory to output plots")
     parser.add_argument("--level",
                         type=str,
                         default="module",
                         choices=["module", "kernel"])
-    parser.add_argument("--top_k",
+    parser.add_argument("--top-k",
                         type=int,
-                        default=9,
+                        default=12,
                         help="Only graph the top `top_k` entries by time.")
-    parser.add_argument("--ignore_sampler",
-                        action='store_true',
-                        help="Ignore everything under the \"Sampler\" module")
+    parser.add_argument("--fold-json-node",
```

Comment:
can you add a quick description of how this works? something like, "aggregate all children of nodes with these names"